<launch>
    <node pkg="robot_localization" type="ekf_localization_node" name="ekf_gps_localization">
        <rosparam command="load" file="$(find husky_gps_nav)/config/gps_localization.yaml" />
        <remap from="odometry/filtered" to="odometry/filtered/global"/> 
<!--        <remap from="odometry/filtered" to="odometry/filtered/global"/> -->

    </node>
</launch>

